Tests/Foundation/UserDefaultsMonitorTests.swift

```diff
@@ -0,0 +1,67 @@
+//
+//  UserDefaultsMonitorTests.swift
+//  XestiMonitors
+//
+//  Created by Paul Nyondo on 2018-04-25.
+//
+//  © 2018 J. G. Pusey (see LICENSE.md)
+//
+
+import XCTest
+@testable import XestiMonitors
+
+internal class UserDefaultsMonitorTests: XCTestCase {
+    let notificationCenter = MockNotificationCenter()
+    let userDefaults = UserDefaults.standard
```

Comment:
I think the tests will be better if you use `UserDefaults()` to create a brand new (non-shared) instance.